jekyll/_cci2/configuration-reference.md

```diff
@@ -1391,6 +1391,181 @@ ignore | N | String, or List of Strings | Either a single tag specifier, or a li
 
 For more information, see the [Executing Workflows For a Git Tag]({{ site.baseurl }}/2.0/workflows/#executing-workflows-for-a-git-tag) section of the Workflows document.
 
+###### **`matrix`**
+A matrix job allows you to run a parameterized job multiple times with different
+arguments.
+
+Key | Required | Type | Description
+----|----------|------|------------
+parameters | Y | Map  | A map of parameter names to every value the job should be called with
+exclude | N | Map | A list of argument maps that should be excluded from the matrix
+alias | N | String | An alias for the matrix, usable from another job's `requires` stanza. Defaults to the name of the job being executed
+
+The following is a basic example of using matrix jobs. All features are
+explained in more detail below.
+
+```yaml
+workflows:
+  workflow:
+    jobs:
+      - build:
+          # In the invocation of a matrix job, you may use the special
+          # template variables `<< matrix.* >>`. These will expand to
+          # the value of the parameter for _each_ job within the matrix.
+
+          # Naming each job in the matrix can be useful if you want to
+          # require a particular matrix job from another job. The name
+          # defaults to the job name ("build" here) concatenated with
+          # all of the matrix values, sorted by their names in
+          # alphabetical order.
+
+          # The default name for this example would be:
+          # name: "build-<< matrix.platform >>-<< matrix.version >>"
+
+          name: "build-v<< matrix.version >>-for-<< matrix.platform >>"
+
+          # You may also require jobs. You can even use this to require a
+          # particular matrix job from another matrix job!
+          requires:
+            - lint
+            - "prepare-v<< matrix.version >>-for-<< matrix.platform >>"
+
+          matrix:
+            # The job will be invoked once for each combination of parameters.
+            # In this case, the job would be invoked 9 times, once for each
+            # version/platform combination.
+            parameters:
+              version: ["0.1", "0.2", "1.0"]
+              platform: ["windows", "linux", "macos"]
+
+            # You may provide a matrix alias in order to require the entire
+            # matrix later. The default alias is the name of the job being
+            # invoked, which in this case would be:
+            # alias: build
+            alias: my-matrix
+
+            # You may `exclude` some sets of parameters from the matrix.
+            exclude:
+              - version: 0.2
+                platform: linux
+              - version: 1.0
+                platform: macos
+```
```

Comment:
To me, this is the "basic example" of matrices:

```yaml
- build:
    matrix:
      parameters:
        version: ["0.1", "0.2", "1.0"]
        platform: ["windows", "linux", "macos"]
```

If you really want a full comprehensive example as well, for people to refresh their memory on the syntax, it would be more on-brand with the rest of our documentation if that came at the end of the matrix section and didn't have as many comments.

Reply:
This makes sense - I removed the "comprehensive example."